cas_client/cas_client.py

```diff
@@ -331,12 +320,12 @@ def session_exists(self, ticket):
     ### PRIVATE METHODS ###
 
     def _build_auth_token_data(
-        self,
-        auth_token_ticket,
-        authenticator,
-        private_key,
-        **kwargs
-        ):
+            self,
+            auth_token_ticket,
+            authenticator,
+            private_key,
+            **kwargs
```

Comment:
/nit why the extra indent here?

Reply:
good catch - `pycharm` did that for me. I'll tweak the config.